packages/blocks/src/components/spinner/index.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import { keyframes } from '@emotion/core';
+import styled from '@emotion/styled';
+
+const spinAnimation = keyframes`
```

Comment:
Nit: Hmm... let's maybe put that into `packages/styles/src/animations.js` so we have one reference for all our generic animations.